f5/bigip/__init__.py

```diff
@@ -43,6 +44,7 @@ def __init__(self, hostname, username, password, **kwargs):
         self._meta_data = {
             'allowed_lazy_attributes': allowed_lazy_attrs,
             'hostname': hostname,
+            'port': port,
             'uri': 'https://%s/mgmt/tm/' % hostname,
```

Comment:
I originally thought that it should, but there is code further down in the library that uses the uri in some ways that I wasn't able to easily grab the port from....or something like that.

Lemme see if I can find my reasoning behind doing that. I think it was in the resource.py file.